documentation/README.md

```diff
@@ -165,7 +165,7 @@ The command will update the _total_ read counts of each sample and check against
 Much like the group of analysis subcommands you can perform an upload of analysis results (stored in Housekeeper) all at once by running:
 
 ```bash
-cg upload --family raredragon
+cg upload --case raredragon
```

Comment:
```suggestion
cg upload --case raredragon
```
ah indeed so it would be case in the cli command for the user.

Reply:
ChatGPT suggests that only "-c" and "--case" will actually set this parameter, while "case_id" would become the variable name. Hence maybe "-c, --case, case_internal_id" would be the most attractive option to go for?